package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "webpack-plugin-hash-output",
-  "version": "1.3.0",
+  "version": "2.0.0",
   "description": "Plugin for Webpack that uses the final output to compute the hash of an asset",
   "main": "src/OutputHash.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "eslint . && mocha --reporter spec test/*.js",
+    "test": "mocha --reporter spec test/*.js && eslint .",
```

Comment:
I changed the order so that the eslint rules failing doesn't block the test from being run.

I had some issues with linebreak-styling which were fixed by running eslint --fix before committing (I can also add pre-commit hook eslint checking if you want).